src/components/Dropdown/Dropdown.tsx

```diff
@@ -39,6 +40,9 @@ import { screenReaderContainerStyles } from '../../lib/accessibility/Styles/acce
 import ListItem from '../List/ListItem'
 
 export interface DropdownProps extends UIComponentProps<DropdownProps, DropdownState> {
+  /** The initial value for the open state. */
+  defaultIsOpen?: boolean
```

Comment:
We just just `open`, without `is` prefix 💭

Reply:
@layershifter 
will do, but it's gonna be a bit confusing in the code because we're mapping it to `isOpen` prop from `Downshift`, just a heads up

Reply:
done